docs/digger/installation/installer-variables.asciidoc

```diff
@@ -0,0 +1,295 @@
+== Variables
+
+Tables below shows all variables used in the installer per role.
+
+=== android-sdk
+
+|===
+| Variable name | Description | Default value
+
+|android_sdk_home
+|Location of Android SDK that is installed in the Android SDK container.
+|/opt/android-sdk-linux
+|===
+
+=== configure-buildfarm
+
+|===
+| Variable name | Description | Default value
+
+|project_name
+|Name of the OpenShift project.
+|jenkins
+
+|concurrent_android_builds
+|Number of max number of concurrent Android builds that are allowed.
+|5
+|===
+
+
+=== deploy-jenkins
+
+|===
+| Variable name | Description | Default value
+
+|project_name
+|Name of the OpenShift project.
+|jenkins
+
+|enable_oauth
+|Whether to enable OAuth OpenShift integration. If false, a static account will be initialized.
+|false
+
+|master_memory_limit
+|Maximum amount of memory the Jenkins master container can use.
+|3Gi
+
+|master_volume_capacity
+|Volume space available for data.
+|40Gi
+
+|===
+
+=== deploy-nagios
+
+Variables below are used in Nagios for sending alert emails.
+
+|===
+| Variable name | Description | Default value
+
+|smtp_server
+|SMTP server to send alert emails.
+|localhost
+
+|smtp_username
+|SMTP username.
+|username
+
+|smtp_password
+|Password of the SMTP user.
+|password
+
+|smtp_from_address
+|SMTP from address.
+|admin@example.com
+
+|rhmap_admin_email
+|Destination address of alert emails.
+|root@localhost
+
+|===
+
+
+=== java
+
+Variables below are to configure the JDK to install on the remote that runs Jenkins CLI commands.
+
+|===
+| Variable name | Description | Default value
+
+|repo
+|Repository to install JDK.
+|rhel-7-server-optional-rpms
+
+|java_version
+|JDK version to install.
+|1.8.0
+
+|===
+
+
+=== login
+
+|===
+| Variable name | Description
+
+|oc_user
+|OpenShift user to login to OpenShift.
+
+|oc_password
+|OpenShift user password.
+
+|===
+
+=== provision-osx
+
+Variables below are used while provisioning an OSX node.
+
+|===
+| Variable name | Description | Default value
+
+|remote_tmp_dir
+|What directory to use when creating some temporary files.
+|/tmp
+
+|nvm_install_url
+|The URL of nvm installation script.
+|https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh
+
+|nvm_install_file_name
+|What to name the nvm installation script on the node.
+|install-nvm.sh
+
+|ruby_version
+|The version of Ruby to install on the node.
+|ruby-2.3.3-latest
+
+|node_versions
+|A list of Node versions to install.
+|6
+
+|xcode_install_version
+|The version of the xcode-install tool to install on the node.
+|2.2.1
+
+|gem_packages
+|A list of gems to install on the node. Format: `{ name: <PACKAGE_NAME>, version: <PACKAGE_VERSION> }`
+|Only installs xcode-install by default.
```

Comment:
We are adding public_suffix gem to default list.
We could have problem if user overrides this list and doesn't include default ones. @PhilipGough wdyt?

Reply:
Agreed, these should be forced

Reply:
@matskiv what change is required here? should we remove it from the docs as we don't want it to be overriden?

Reply:
@laurafitzgerald yes, I guess removing it is safest option. We can later add different list for user defined gems.